src/utilities/__tests__/findBreakingChanges-test.js

```diff
@@ -37,12 +37,14 @@ describe('findBreakingChanges', () => {
     const newSchema = buildSchema(`
       type Type2
     `);
-    expect(findBreakingChanges(oldSchema, newSchema)).to.deep.equal([
-      {
-        type: BreakingChangeType.TYPE_REMOVED,
-        description: 'Type1 was removed.',
-      },
-    ]);
+    expect(findBreakingChanges(oldSchema, newSchema)[0]).to.deep.include({
+      type: BreakingChangeType.TYPE_REMOVED,
+      description: 'Type1 was removed.',
+    });
+    // flow ensures that oldNode is of type ASTNode, checking its presence should be enough
+    expect(findBreakingChanges(oldSchema, newSchema)[0]).to.have.property(
+      'oldNode',
+    );
```

Comment:
You don't need snapshot here just check that node inside the change is the same as in in-memory schema:

```js
expect(findBreakingChanges(oldSchema, newSchema)).to.deep.equal([
  {
     type: BreakingChangeType.TYPE_REMOVED,
     description: 'Type1 was removed.',
     oldNode: oldSchema.getType('Type1').astNode,
     newNode: undefined,
  },
]);
``
```

Reply:
Thank you, that is a smart way to solve my issue indeed. I will rework those tests this way!